util/tokencount/tokctnightly

```diff
@@ -47,7 +47,7 @@ $user = `whoami`;
 chomp($user);
```

Comment:
I ended up leaving this in because I either wanted to blow away this whole directory or none of it, but didn't want to take the time to review what's in it right now.

Reply:
Yeah, makes sense.